basics/signals/src/button.ts

```diff
@@ -0,0 +1,35 @@
+import { Widget } from '@lumino/widgets';
+import { ISignal, Signal } from '@lumino/signaling';
+
+export interface ICount {
+  clickCount: number;
+}
+
+export class ButtonWidget extends Widget {
+  constructor(options = { node: document.createElement('button') }) {
+    super(options);
+
+    this.node.textContent = 'Clickme';
+
+    /**
+     * The class name, jp-ButtonWidget, follows the CSS class naming
+     * convention for classes that extend lumino.Widget.
+     */
+    this.addClass('jp-ButtonWidget');
```

Comment:
Could you extract the class name as constant to align with JupyterLab practice?

Reply:
Fixed as requested.